README.md

```diff
@@ -1,5 +1,20 @@
 # Lustre Collector
 
+[![Build Status](https://travis-ci.com/whamcloud/lustre-collector.svg?branch=master)](https://travis-ci.com/whamcloud/lustre-collector)
+
 This repo provides data fetching and extraction capabilities for Lustre statisics.
 
-[![Build Status](https://travis-ci.com/whamcloud/lustre-collector.svg?branch=master)](https://travis-ci.com/whamcloud/lustre-collector)
+It is provided as a standalone binary that can be called to retrieve stats in the desired output.
+
+## Usage
+
+```bash
+# Will return OST related stats in JSON format
+lustre_collector --host=oss1.domain --target_type=oss
```

Comment:
Couple of questions:
1. Is `host` a required argument? I'm assuming it would be otherwise it wouldn't know where to check right?
2. If you don't pass a target type would you get data for mgs, oss, and mds?
3. If you don't specify a format does it default to JSON?

Reply:
> Is host a required argument? I'm assuming it would be otherwise it wouldn't know where to check right?

This runs local on the storage server. It hasn't been implemented yet, but I think it will default to `hostname` if not provided.

> If you don't pass a target type would you get data for mgs, oss, and mds?

Given it runs local on the storage server, it will collect the relevant data for that server. There might be an easier way to sniff out the type of server and return based on that, but for now I'm just making it hardcoded.

> If you don't specify a format does it default to JSON

Yes, defaults to JSON.